packages/components/src/DataTable/Body.tsx

```diff
@@ -2,13 +2,22 @@ import { Row, Table, flexRender } from "@tanstack/react-table";
 import classNames from "classnames";
 import React, { useCallback } from "react";
 import styles from "./DataTable.css";
+import { Heading } from "../Heading";
+import { Text } from "../Text";
 
 interface BodyProps<T> {
   table: Table<T>;
   onRowClick?: (row: Row<T>) => void;
+  height?: number;
+  emptyTableBodyMessage?: string;
```

Comment:
Are we choosing to limit this child element to text only? It seemed like based on Chris's comment there could be other types of content we may wish to provide (e.g. "an illustrative element")

Reply:
@iremdesign

Reply:
That's correct, my suggestion was intended to be that this prop should actually accept a child component rather than a string – this way, a wider range of use cases can be supported for individual teams and we don't need to be overly prescriptive at the system level.